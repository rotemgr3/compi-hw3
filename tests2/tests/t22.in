 
//piazza 88 - should be sintax error bc of string x +
override int foo(int y) {int booga = 5;}
override int foo(int y, string x) {int booga = 5;}
void main() {int booga = 5;}
