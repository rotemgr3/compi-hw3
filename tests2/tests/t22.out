---end scope---
y INT -1
booga INT 0
line 4: syntax error
